sdk/samples/aztables/sample_create_delete_table.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/Azure/azure-sdk-for-go/sdk/data/aztables"
+)
+
+func CreateFromServiceClient() {
+	cred, err := azidentity.NewDefaultAzureCredential(nil)
+	check(err)
+	accountName, ok := os.LookupEnv("TABLES_STORAGE_ACCOUNT_NAME")
```

Comment:
The boilerplate is starting to obscure what you're doing.

Can you consolidate and just have two functions (one for table client, one for service client) and just do the Create/Delete in each of those, rather than separating it out into 4?